### ISSUE TITLE
Lacks initiative

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#5851]
**Preferred term label:**
Lacks initiative

**Synonyms**
lacking in initiative
lack of initiative

**Definition (free text, please give PubMed ID)**
Lacking the ability to decide what to do next and to do it, without needing other people to tell you what to do. 
Source: https://www.collinsdictionary.com/us/dictionary/english/initiative#:~:text=uncountable%20noun,energy%20More%20Synonyms%20of%20initiative

**Parent term (use hpo.jax.org/app)**
HP_0031466 'Impairment in personality functioning'

This is needed for the Kids First INCLUDE supplement.

**Comment by pnrobinson** [comment]
@nicolevasilevsky can your colleagues comment on how this relates to https://hpo.jax.org/app/browse/term/HP:0000745

**Comment by nicolevasilevsky** [comment]
I think we could use that term - thanks Peter!

If you don't mind, I'll add these as synonyms.

**Comment by pnrobinson** [comment]
thanks, just added the synonyms as LPs!
